packages/lesswrong/lib/vendor/react-click-away-listener.ts

```diff
@@ -35,14 +35,12 @@ import React, {
   FunctionComponent
 } from 'react';
 
-type FocusEvents = 'focusin' | 'focusout';
```

Comment:
When Sarah and I were testing we ran into some cases where focus events would wrongly trigger the clickaway. I'm not sure if these would still happen with the other bug fixes, but I don't think we benefit from having clickaways triggered on focus events anywhere so I decided to remove them to avoid future problems